episodes/00-sql-introduction.md

```diff
@@ -112,9 +112,14 @@ which is stored as one of its fields.
 Records may also contain keys that refer to records in other tables,
 which enables us to combine information from two or more sources.
 
-
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+The *relational* concept is connected with organizing information across multiple tables in the database.
```

Comment:
The phrase "is connected with" feels vague, and I'm not sure that this sentence adds anything beyond what's in 118. I'd consider omitting it.